crml/staking/src/lib.rs

```diff
@@ -1373,6 +1373,11 @@ impl<T: Trait> Module<T> {
 		imbalance
 	}
 
+	#[cfg(any(feature = "std", test))]
+	pub fn get_current_era_transaction_fee_reward() -> BalanceOf<T> {
+		CurrentEraTransactionRewards::<T>::get()
+	}
```

Comment:
Not sure if this is frowned upon to have a function guarded behind test. But there are some examples from plug here:: https://github.com/plugblockchain/plug-blockchain/blob/develop/frame/system/src/lib.rs#L853

Reply:
I agree with  @cowboy-bebug, but cannot find a nice way of exposing this without a public getter.
This will have to do unless we find a better solution